udata/migrations/2023-02-08-rename-internal-dates.py

```diff
@@ -0,0 +1,46 @@
+'''
+The purpose here is to change the dates name in the dataset and resource model.
+'''
+import logging
+
+from mongoengine.connection import get_db
+from udata.models import Dataset
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing Dataset collection.')
+
+    db = get_db()
+    dataset_collection = db.dataset
+
+    result = dataset_collection.update_many({}, {'$rename': {
+        'created_at': 'created_at_internal',
+        'last_modified': 'last_modified_internal'}})
+    log.info(f'{result.modified_count} Datasets processed.')
+
+    community_resource_collection = db.community_resource
+    result = community_resource_collection.update_many({}, {'$rename': {
+        'created_at': 'created_at_internal',
+        'modified': 'last_modified_internal'}})
+    log.info(f'{result.modified_count} Community Resources processed.')
+
+    result = dataset_collection.update_many({}, [{
```

Comment:
second argument can't be an array, can it?

I have the following error at this step when running `udata db migrate`:
```
pymongo.errors.OperationFailure: BSON field 'update.updates.u' is the wrong type 'array', expected type 'object', full error: {'ok': 0.0, 'errmsg': "BSON field 'update.updates.u' is the wrong type 'array', expected type 'object'", 'code': 14, 'codeName': 'TypeMismatch'}
  ```